### PR TITLE
feat(MintingTokens): Created welcome view

### LINKED ISSUE #8732: [Community Mint Collectibles] Create welcome page
### Description
[Epic](https://github.com/status-im/status-desktop/issues/8694)

[Welcome page design](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=22721%3A498587&t=CIOm9ph09U1GV7tf-0) 

Screenshot 2022-12-12 at 12 46 46

**NOTE:** Use same qml structure than used in `Permissions` for navigation between views for code consistency.

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #8732

### What does the PR do

- Created base welcome settings view (replaced specific component used for permissions and parametrised to be used as the base class for some of the next community settings functionalities (at least for permissions, minting and airdop)).
- Created minting tokens welcome view: Added basic navigations and added storybook support for it.

KNOWN ISSUE: Image in dark mode is not displayed correctly. We are waitting for the designers to provide us an image without backgroud.

NOTE: `Create new collectible` view is not part of this task and it is just a mocked / test UI.

### Affected areas

- Existing Community Permissions Welcome page
- New Minting Tokens Welcome page

### Screenshot of functionality

https://user-images.githubusercontent.com/97019400/219648940-6c224027-f6e9-4701-8b7b-5d32832f2b35.mov

### CODE REVIEW

**Review by alexandraB99 — APPROVED** [review]
se ve bien, gracias! :D

File: ui/app/AppLayouts/Chat/views/communities/CommunityWelcomeSettingsView.qml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+                ColumnLayout {
+                    id: checkersColumn
+
+                    property int rowChildSpacing: 10
+                    property color rowIconColor: Theme.palette.primaryColor1
+                    property string rowIconName: "checkmark-circle"
+                    property int rowFontSize: 15
+                    property color rowTextColor: Theme.palette.directColor1
+                    property double rowTextLineHeight: 1.2
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/views/communities/CommunityWelcomeSettingsView.qml:81]: `readonly` those props?

**Review by caybro — COMMENTED** [review]
Looks nice, minor things inline

Since I can't add comments to an image, what about a dark version of `ui/imports/assets/png/community/mint2_1.png`?

**Comment by noeliaSD** [comment]
> 

Yeah, we are waiting for an images without background. In the next days the design team will provide it and I will update it accordingly!

**Review by endulab — APPROVED** [review]

**Review by caybro — APPROVED** [review]

**Pull request merged**